src/components/app/CardsContext.js

```diff
@@ -1,21 +1,19 @@
-import React, {useState} from "react";
+import React, {useEffect, useState} from "react";
 import {v4 as uuidv4} from "uuid";
+import axios from "axios";
 
 export const CardsContext = React.createContext(null)
 
 function CardsProvider({children}) {
-    const defaultData = {header: 'Caption', body: 'Text...'};
+    const [cards, setCards] = useState([]);
+    useEffect(() => {
+        axios('https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json')
+            .then(response => setCards(response.data.slice(0, 15).map(obj => {
+                return {id: uuidv4(), header: obj['Name'], body: obj['About']}
+            })))
+            .catch(reject => console.log(`Something went wrong: ${reject}`))
```

Comment:
Actually could be a bit simplified:
```
useEffect(() => {
    axios('https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json')
        .then(response => setCards(
            response.data.slice(0, 15).map(obj => ({
                id: uuidv4(), header: obj['Name'], body: obj['About']
            }))))
        .catch(reject => console.log(`Something went wrong: ${reject}`))
}, [])
```
Also, if you use a proper tabulation - it increases code readability. You can use some linter package or IDE plugin for such purposes (prettier, etc.)